src/backend/utils/adt/jsonpath_exec.c

```diff
@@ -578,6 +592,14 @@ tsql_openjson_with_columnize(Jsonb *jb, char *col_info)
 	/* get tuple set using executeJsonPath */
 	jp = DatumGetJsonPathP(DirectFunctionCall1(jsonpath_in, CStringGetDatum(col_path)));
 
+	/*
+	 * In case where there is no space between 'strict/lax' and the column path
+	 * jsonpath_in will automatically set the header to strict. We need to change
+	 * the header to prevent unintended 'strict' calls to openjson
+	 */
+	if(!strict)
+		jp->header = JSONPATH_LAX + 1;
```

Comment:
Or more likely, just use `jp->header |= JSONPATH_LAX`, unless that is not giving the desired behavior

Reply:
Yeah `jp->header |= JSONPATH_LAX` is the same behavior im looking for, I can update the code. The point is to get it to pass a couple check statements such as this in the `executeJsonPath` function
`cxt.laxMode = (path->header & JSONPATH_LAX) != 0;` which will set it to lax mode rather than keeping it as strict, and this assert in the `jspInit` function `Assert((js->header & ~JSONPATH_LAX) == JSONPATH_VERSION);`